sklearn/gaussian_process/gpr.py

```diff
@@ -209,12 +222,17 @@ def obj_func(theta, eval_gradient=True):
                         "Multiple optimizer restarts (n_restarts_optimizer>0) "
                         "requires that all bounds are finite.")
                 bounds = self.kernel_.bounds
-                for iteration in range(self.n_restarts_optimizer):
+                theta_initials = []
+                for i in range(self.n_restarts_optimizer):
                     theta_initial = \
                         self.rng.uniform(bounds[:, 0], bounds[:, 1])
-                    optima.append(
-                        self._constrained_optimization(obj_func, theta_initial,
-                                                       bounds))
+                    theta_initials.append(theta_initial)
+                Parallel(n_jobs=self.n_jobs)(delayed(self._optima_iterations,
```

Comment:
there are a few ways you could make this line wrapping more readable! please find one...